driver/src/contracts/stablex_contract.rs

```diff
@@ -44,6 +47,60 @@ pub trait StableXContract {
     ) -> Result<()>;
 }
 
+struct AuctionElement {
+    valid_from: U256,
+    valid_until: U256,
+    order: Order,
+}
+
+impl AuctionElement {
+    fn from_bytes(bytes: &[u8; 113]) -> Self {
+        let mut data_vector = bytes.to_vec();
+        println!("{}", bytes.len());
+        let account_id = H160::pop_from_log_data(&mut data_vector);
+        // TODO - not sure what this is for.
+        let _sell_token_balance = U256::pop_from_log_data(&mut data_vector);
```

Comment:
It's the amount of tokens the user has currently in their account (for the `balance > 0` constraint).

Reply:
Right, and the remaining amount is the amount left in their order I suppose.